Cole/BFB-336: Added Pistol Asset

csweet22 opened the pull request:
**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-336

**Description**
Added pistol asset to project.

All new files can be found under:
Assets > _BForBoss > _Weapons > Pistol

Example of the asset can be found in MT_Weapon_Sandbox on the spawn point platform.

**Screenshots / GIFs**
![image](https://user-images.githubusercontent.com/94721502/173198801-f7bcd6de-f7e6-4e67-b54f-a9222f19c7a1.png)

csweet22 commented:
Just let me know if you want me to add a commit where the gun is not in the MT_Weapon_Sandbox scene before approving. 👍

m4rtint commented:
Can you put it into PG_Items_Library. 

Or if you have your own scene with all the assets.

csweet22 commented:
> Can you put it into PG_Items_Library.
> 
> Or if you have your own scene with all the assets.

I've removed the gun from MT_Weapon_Sandbox and added it to PG_Items_Library 👍
